.github/workflows/npm-release.yml

```
@@ -0,0 +1,63 @@
+name: npm-release
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        type: choice
+        description: "Select version increment type (follows Semantic Versioning)"
+        required: true
+        options:
+          - patch
+          - minor
+          - major
+      app-path:
+        type: choice
+        description: "Select the node service to release"
+        required: true
+        options:
+          - refarch-frontend # Add or modify frontend paths according to your project structure
+jobs:
+  release:
+    name: Release
```

```suggestion
    name: release
```

---

theoretisch muss man jetzt alles nochmal testen, ob die ganzen output variablen noch funktionieren. bei den ganzen groß und kleinschreibung sachen

---

```suggestion
    name: release
```
english capitalization